yarn.lock

```diff
@@ -1273,6 +1273,14 @@
   resolved "https://registry.yarnpkg.com/@colors/colors/-/colors-1.5.0.tgz#bb504579c1cae923e6576a4f5da43d25f97bdbd9"
   integrity sha512-ooWCrlZP11i8GImSjTHYHLkvFDP48nS4+204nGb1RiX/WXYHmJA2III9/e2DWVabCESdW7hBAEzHRqUn9OUVvQ==
 
+<<<<<<< HEAD
```

Comment:
hmm, you have a merge conflict marker added to the `yarn.lock` file, you might need to regenerate the yarn.lock file to fix it

Reply:
Added gitIgnore to ignore all yarn files.